gzip.mli

```diff
@@ -118,6 +118,10 @@ val flush: out_channel -> unit
            dispose of the resources associated with the compression
            channel.  This can be useful if e.g. the underlying file channel
            is a network socket on which more data is to be sent. *)
+val flush_continue: out_channel -> unit
+       (** Flush data data through both the compression channel and the
```

Comment:
Typo in "Flush data data".  What about: "Flush all pending compressed data through both ..." ?

Reply:
Good spot - thanks for the correction